neore/scripts/package.sh

```diff
@@ -67,6 +67,10 @@ function download_flume(){
     rm -f ${FLUME_WO_TAR}/lib/libthrift-*.jar
     mv ${LIBTHRIFT_JAR} ${FLUME_WO_TAR}/lib
 
+    _log "### Disable httpclient and httpcore libraries distributed within apache-flume bundle... ###"
+    mv ${FLUME_WO_TAR}/lib/httpclient-4.2.1.jar ${FLUME_WO_TAR}/lib/httpclient-4.2.1.jar.old
```

Comment:
Link to sources: https://github.com/telefonicaid/fiware-cygnus/blob/develop/doc/installation_and_administration_guide/install_from_sources.md#installing-apache-flume

Suggestion: to add the steps to remove/rename the .jar files in the commands script shown at that page, using the explantory test to explain the nature of these remove/rename commands.